examples/contracts/src/lib.rs

```diff
@@ -79,44 +78,8 @@ mod tests {
     }
 
     #[tokio::test]
-    async fn manual_storage_init() -> Result<(), Error> {
-        use fuels::prelude::*;
-
-        abigen!(
-            MyContract,
-            "packages/fuels-abigen-macro/tests/test_projects/storage/out/debug/storage-abi.json"
-        );
-
-        let wallet = launch_provider_and_get_wallet().await;
-
-        // ANCHOR: storage_slot_create
-        let storage_slot = create_storage_slot("slot", 42);
-        // ANCHOR_END: storage_slot_create
-
-        let key = **storage_slot.key();
-        let expected_value = storage_slot.value().to_owned();
-
-        // ANCHOR: manual_storage
-        let contract_id = Contract::deploy(
-            "../../packages/fuels-abigen-macro/tests/test_projects/storage/out/debug/storage.bin",
-            &wallet,
-            TxParameters::default(),
-            vec![storage_slot],
-        )
-        .await?;
-        // ANCHOR_END: manual_storage
-
-        let contract_instance = MyContract::new(contract_id.to_string(), wallet.clone());
-
-        let value = contract_instance.get_value(key).call().await?.value;
-        assert_eq!(expected_value.as_slice(), value.as_slice());
-
-        Ok(())
-    }
-
-    #[tokio::test]
-    // ANCHOR: deploy_with_salt
-    async fn deploy_with_salt() -> Result<(), Error> {
+    // ANCHOR: deploy_with_parameters
```

Comment:
I think you removed a commit by mistake

Reply:
After some changes the test in `examples` turned out to be the same as the one in `harness`. I moved the ANCHOR tags to `harness` and removed the example.